src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -886,7 +901,7 @@ internal RunVersionOne CreateRunVersionOne(Run v2Run)
 
                     run.BaselineId = v2Run.BaselineGuid;
                     run.Files = CreateFileDataVersionOneDictionary();
-                    run.Id = v2Run.AutomationDetails?.Guid;
+                    run.Id = v2Run.AutomationDetails?.Id;
```

Comment:
>Id [](start = 54, length = 2)

I don't think any other customer is both producing v1 and using automation details. I think this is the right value for `run.Id` given the decision below for `run.stableId`.